app/plugins/music_service/mpd/index.js

```diff
@@ -1007,7 +1008,13 @@ ControllerMpd.prototype.createMPDFile = function (callback) {
                 var conf12 = conf11.replace('"${ffmpeg}"', 'decoder { ' + os.EOL + 'plugin "ffmpeg"'  + os.EOL + 'enabled "yes"'  + os.EOL + 'analyzeduration "1000000000"'  + os.EOL + 'probesize "1000000000"' + os.EOL + '}' + os.EOL);
 			} else {
                 var conf12 = conf11.replace('"${ffmpeg}"', " ");
-			}
+            }
+            
+            for(var callback of self.registeredCallbacks)
+            {
+               var data = self.commandRouter.executeOnPlugin(callback[0], callback[1], callback[2]);
```

Comment:
First problem is that this will work only with 1 callback. So please revise the for loop.

Second
Please don't use a straight array like this, as it's not really clear. Instead use an object. It will become like this:
var data = self.commandRouter.executeOnPlugin(callback.type, callback.plugin, callback.method, callback.data);
This will help developers understand what they need to put, and eventually also add a variable to pass to the function.